cyborg/bin/check-pull-request.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 Greptime Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as core from '@actions/core'
+import {handleError, obtainClient} from "@/common";
+import {context} from "@actions/github";
+import {PullRequestEvent} from "@octokit/webhooks-types";
+import {Options, sync as conventionalCommitsParser} from 'conventional-commits-parser';
+import conventionalCommitTypes from 'conventional-commit-types';
+import _ from "lodash";
+
+const defaultTypes = Object.keys(conventionalCommitTypes.types)
+const breakingChangeLabel = "breaking-change"
```

Comment:
Once this patch accepted, I'll rename the current `breaking change` label with `breaking-change`. See #3849 for more details.